packages/legend-application-studio/src/components/editor/editor-group/mapping-editor/DEPRECATED__MappingTestEditor.tsx

```diff
@@ -735,8 +737,13 @@ export const DEPRECATED__MappingTestEditor = observer(
       flowResult(testState.runTest()),
     );
     const cancelTest = (): void => {
-      testState.setIsRunningTest(false);
-      testState.setTestRunPromise(undefined);
+      flowResult(testState.cancelTest()).catch((error) => {
```

Comment:
lets inline the try, catch logic within the `cancelTest` method and the other cancel methods